README.md

```diff
@@ -53,28 +77,24 @@ After following the steps, run the command below and follow the instructions to
 ts-node src/google/create-token.ts
 ```
 
-## Deployment
+## Development
 
-Before deployment, create an `.env` file using the fields mentioned below:
+To add new features to the bot. Fork the repository and add your feature to `src/meetbot/features`. After finishing your changes, [run the bot locally](#running-the-bot-locally) to check if your feature loads and test your changes. 
 
-```
-GOOGLE_PASSWORD=              # Password for your bot/user that is used to join Google Meet. 
-GOOGLE_TOTP_SECRET=           # [Optional] If the bot account has 2FA auth, then add 2FA secret here.
-GOOGLE_LOGIN=                 # Email-ID/username of your bot/user
-TOKEN_PATH=token.json         # [Default] Path to token file 
-HTTP_PORT=80                  # Port to run express server on. For prod, use port 80.
-RUNNING_ALPINE=0              # [Default] If base image is alpine then set the env var, otherwise don't. 
-```
+To develop the UI, run `npm run dev-ui`. This will spin up the development server. You should then be able to access the dashboard at `http://localhost:3000/`. Note that the data you will see on the tables are actually being mocked by MirageJS, and not actual request is going to the API. This makes it easier to develop both components in an entirely decoupled way.
 
-Run the command below and follow the instructions to generate a `token.json` file. This authentication process is one time only.
+## Deployment
 
-```
-ts-node src/google/create-token.ts
-```
+To activate all the features of meetbot, [authenticate the meetbot](#authentication) and create the `token.json` file before deployment. 
 
-To deploy, use the dockerfiles present in the repository. Tweak the base image and other details as per your deployment platform (AWS, Heroku etc.) and deploy. 
-To deploy on balenaCloud, run the command below:
+To deploy on balenaCloud, use the balena CLI and balena push command as stated below. For more information [check out the docs](https://www.balena.io/docs/learn/deploy/deployment/).
 
 ```
 balena push <Name of fleet>
 ```
+
+To deploy on a server, create and fill the `.env` file in the root of the project directory using the `.env.example` file and deploy using the Dockerfile present in the root directory of this repository. 
+
+## Credits
+
+Meetbot's google meet voice caption capture is implemented using [dzaman/google-meet](https://github.com/dzaman/google-meet-transcripts)'s stenographer implementation licensed under GNU GPLv3.
```

Comment:
really? That means we can't place our code under the Apache license

Reply:
Another thing to discuss. I saw this and we need to figure it out. Can stenographer be selectively GPLv3?

Reply:
Lawyers can discuss *ages* about this. But the FSF is pretty clear on that: As soon as you include GPL code, your code must be under a compatible license. Which in this case would only be GPL or AGPL